docs/manual/introduction/Browser-support.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<base href="../../" />
+	<script src="list.js"></script>
+	<script src="page.js"></script>
+	<link type="text/css" rel="stylesheet" href="page.css" />
+</head>
+<body>
+	<h1>[name]</h1>
+
+	<h2>Overview</h2>
+	<div>
+		<p>
+			Three.js can use WebGL to render your scenes on all modern browsers. For older browsers, especially Internet Explorer 10 and below, you may have to fallback to one of the other [link:https://github.com/mrdoob/three.js/tree/master/examples/js/renderers renderers] (CSS2DRenderer, CSS3DRenderer, SVGRenderer, CanvasRenderer). Additionally, you may have to include some polyfills, especially if you are using files from the [link:https://github.com/mrdoob/three.js/tree/master/examples /examples] folder.
+		</p>
+		<p>
+			Note: if you don't need to support these old browsers, then it is not recommended to use the other renderers as they are slower and support less features than the WebGLRenderer.
+		</p>
+	</div>
+
+	<h2>Browsers that support WebGL</h2>
+	<div>
+		<p>
+			Google Chrome 9+, Firefox 4+, Opera 15+, Safari 5.1+, Internet Explorer 11 and Microsoft Edge. You can find which browsers support WebGL at [link:https://caniuse.com/#feat=webgl Can I use WebGL].
+		</p>
+	</div>
+
+	<h2>JavaScript Language Features or Web APIs Used in three.js</h2>
```

Comment:
Sorry, I do not think this section provides any valuable information, and it can be removed.

Reply:
I don't think I agree. I think it's a good start and we can complete it over time. I think this is good information for new people to start grasping how the library works.